File: SWEA/1936.py
# 1대1 가위바위보 / D1

# 가위 1, 바위 2, 보 3
A, B = map(int, input().split())
r = 'A' if A==1 and B==3 or A==2 and B==1 or A==3 or B==2 else 'B'
print(r)

File: SWEA/1936.java
package java_algorithm;

import java.io.IOException;
import java.io.BufferedReader;
import java.io.InputStreamReader;

public class Solution {

	public static void main(String[] args) throws IOException {
		BufferedReader br = new BufferedReader(new InputStreamReader(System.in));
		String[] arr = br.readLine().split(" ");
		int a = Integer.parseInt(arr[0]), b = Integer.parseInt(arr[1]);
		if((a==1&&b==3) || (a==2&&b==1) || (a==3&&b==2)) {
			System.out.println('A');
		}else {
			System.out.println('B');
		}
	}
}